model/testing/src/icon4py/model/testing/datatest_utils.py

```diff
@@ -81,22 +79,6 @@
 }
 
 
-def get_test_data_root_path() -> pathlib.Path:
```

Comment:
this logic was duplicated in definitions and there are more duplications in this file that I would target in a next PR